includes/api/class-give-api-v2.php

```diff
@@ -98,28 +98,31 @@ public function register_routes() {
 		register_rest_route(
 			$this->rest_base,
 			'/form/(?P<id>[\d]+)',
-			array(
-				'methods'  => 'GET',
-				'callback' => array( $this, 'get_forms_data' ),
-			)
+			[
+				'methods'             => 'GET',
+				'callback'            => [ $this, 'get_forms_data' ],
+				'permission_callback' => __return_true(),
```

Comment:
@henryholtgeerts being a callback, shouldn't this be assigned `'__return_true'` (string), where as the above would execute and pass `true` (boolean).

Reply:
Ah yes - good call! Replaced with `__return_true`: https://github.com/impress-org/givewp/pull/5115/commits/557c09ec08eb3e902cd9be8ceacf160cfb64c6b3